express-validator/express-validator.d.ts

```diff
@@ -85,39 +108,44 @@ declare namespace ExpressValidator {
 		/**
 		 * isNumeric accepts zero padded numbers, e.g. '001', isInt doesn't
 		 */
-		isInt(): Validator;
+		isInt(options?: MinMaxOptions): Validator;
 		isLowercase(): Validator;
 		isUppercase(): Validator;
 		isDecimal(): Validator;
 		/**
 		 * Alias for isDecimal
 		 */
 		isFloat(): Validator;
+		isFullWidth(): Validator;
+		isHalfWidth(): Validator;
+		isVariableWidth(): Validator;
 		/**
 		 * Check if length is 0
 		 */
-		notNull(): Validator;
+		//notNull(): Validator; // I don't see this in the documentation or code anywhere.
```

Comment:
From the express-validator readme.txt: "An express.js middleware for node-validator."   
The link to node-validator goes to **validator**: https://github.com/chriso/validator.js
but there is actually a **node-validator** repo authored by the express-validator creator located here: https://github.com/ctavan/node-validator
This library contains all of the items you commented out.  I suspect express-validator is actually based on this fork and these should still be included.  @ctavan to confirm
Can you please uncomment these as this would be a breaking change?